cmd/cuid2/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"github.com/akshayvadher/cuid2"
+	"github.com/urfave/cli/v2"
+	"log"
+	"os"
+)
+
+func main() {
+	app := &cli.App{
+		Name:  "cuid2",
+		Usage: "Create CUID2",
+		Flags: []cli.Flag{
+			&cli.IntFlag{
+				Name:  "n",
+				Value: 1,
+				Usage: "Numbers of ids to generate",
+			},
+			&cli.IntFlag{
+				Name:  "len",
+				Value: 24,
+				Usage: "Length of the Id (between 2 and 32)",
+				Action: func(ctx *cli.Context, v int) error {
+					if v > 32 || v < 2 {
+						return fmt.Errorf("len %v should be between 2 and 36", v)
```

Comment:
len %v should be between 2 and **32**

Reply:
This is validation, so it is checking reverse. What is the question again?

Reply:
Oh error message

Reply:
Done. Please check